fapi-client/src/main/scala/com/gu/facia/api/utils/FaciaContentUtils.scala

```diff
@@ -136,7 +136,7 @@ object FaciaContentUtils {
   )
   def maybeShortUrl(fc: FaciaContent) = fieldsGet(fc)(_.flatMap(_.shortUrl))
   def shortUrl(fc: FaciaContent): String = maybeShortUrl(fc).getOrElse("")
-  def shortUrlPath(fc: FaciaContent) = maybeShortUrl(fc).map(_.replace("http://gu.com", ""))
+  def shortUrlPath(fc: FaciaContent) = maybeShortUrl(fc).map(_.replaceFirst("^[a-zA-Z]+://gu.com", ""))
```

Comment:
What @dominickendrick said. And what Adam said more.
Although I think it doesn't matter now that the `discussionKey` is going to be coming through?